content/applications/finance/payment_acquirers/ogone.rst

```diff
@@ -5,84 +5,107 @@ Ogone
-- :ref:`API User ID <ogone/api_user>`: The ID solely used to identify the user with Ogone.
-- :ref:`API User Password <ogone/api_user>`: Value used to identify the user with Ogone.
-- :ref:`SHA Key IN <ogone/sha_key_in>`: Key used in the signature Odoo send to Ogone.
-- :ref:`SHA Key OUT <ogone/sha_key_out>`: Key used in the signature Ogone send to Odoo.
-
-You can copy your credentials from your Ogone account, and paste them in the related fields under
-the **Credentials** tab.
-
-.. _ogone/api_user:
-
-API User ID and Password
-~~~~~~~~~~~~~~~~~~~~~~~~
-
-If you already created a user and have both its ID and password, just copy them. You can also
-generate a new password from :menuselection:`Configuration --> Users --> Your chosen user --> change
-password`.
-
-If you don't have a user, create one by going to :menuselection:`Configuration --> Users -->
-New User`. Set your **User ID** to get your **password** when you save your new user.
-
-.. image:: ogone/ogone_new_user.png
-   :align: center
-   :alt: Get your password when you save the new user.
-
-.. _ogone/sha_key_in:
-
-SHA Key IN
-~~~~~~~~~~
-
-In order to retrieve the SHA Key IN, log into your ogone account, go to
-:menuselection:`Configuration --> Technical Information --> Data and origin verification -->
-Checks for e-Commerce & Alias Gateway`, and retrieve **SHA Key IN**.
-
-.. _ogone/sha_key_out:
-
-SHA Key OUT
-~~~~~~~~~~~
-
-In order to retrieve the SHA Key OUT, log into your ogone account, go to
-:menuselection:`Configuration --> Technical Information --> Transaction feedback --> All transaction
-submission modes`, and get or generate your **API Key** and **Client Key**. Be careful to copy your
-API key as you'll not be allowed to get it later without generating a new one.
+   - `Ogone's documentation <https://epayments-support.ingenico.com/get-started/>`_.
+
+Settings in Ogone
+-----------------
+
+Create an API user
+~~~~~~~~~~~~~~~~~~
+
+Log into your Ogone account and head to the :guilabel:`Configuration` tab.
+
+You need to create an **API user** to be used in the creation of transactions from Odoo. While you
+can use your main account to do so, using an **API user** ensures that if the credentials used in
+Odoo are leaked, no access to your Ogone configuration is possible. Additionally, passwords for
+**API users** do not need to be updated regularly, unlike normal users.
+
+To create an **API user**, go to :menuselection:`Configuration --> Users` and click on
+:guilabel:`New User`. The following fields must be configured:
+
+.. _ogone/ogone:
+
+- :guilabel:`UserID`: you can choose anything you want.
+- :guilabel:`User's Name, E-mail and Timezone`: you can enter the information you want.
+- :guilabel:`Profile`: should be set to :guilabel:`Admin`.
+- :guilabel:`Special user for API`: should be checked.
+
+After the creation of the user, you are required to generate a password. Save the password and
+**UserID**, as they will be required later on during the setup.
+
```

Comment:
In accordance with @AntoineVDV, add a tip here along the lines of: 
"If your user is already set up, make sure that it is activated without any errors. If not, simply click the "Activate(Errors)" button to reset your user."
We spent some time fixing the Ogone test account and that was one of the issues. It shows up on the Ogone account in Configuration - Users and then in the last column of the table containing the users.